entrypoint.sh

```diff
@@ -41,7 +37,20 @@ git config --global --add safe.directory $GITHUB_WORKSPACE
 
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-echo ktlint version: "$(ktlint --version)"
+ktlint_version=$(ktlint --version)
+echo "ktlint version: $ktlint_version"
+
+if [ "$(printf '%s\n' "0.49.0" "$ktlint_version" | sort -V | head -n1)" = "0.49.0" ]; then
+  if [ "$INPUT_ANDROID" = true ]; then
+    export ANDROID="--code-style=android_studio"
+  else
+    export ANDROID="--code-style=intellij_idea"
+  fi
```

Comment:
It depends... Right now, it is a real default value. But later they will make another by default - their own ruleset, something like a mix of Kotlin and Android conventions. So it will break interoperability. To prevent that, I have specified the current default for future updates

Reply:
Paragraph "'Ktlint Official` code style and renaming of existing code styles" in ktlint changelog you can find in PR desc.

Reply:
Alright... we can only hope that they don't come up with more breaking changes.

Reply:
@berhram @ghaiszaher Unfortunately that breaks code styling for android. Apparently `--code-style=android_studio` is incorrect and it should still be `--android`

Reply:
Thanks @berhram ! In fact that was problem in our editorconfig. We did not specify `ktlint_code_style` in `.editorconfig` at all and relied only on `--android` parameter (which seemed to work fine pre 0.49.0). 

Now after adding 

```
[*.{kt, kts}]
// ...
ktlint_code_style = android_studio
```
it is totally fine

Reply:
Preparing a revert: https://github.com/ScaCap/action-ktlint/pull/41
`--android` seems to be working fine with latest version.